tableau_blueprints/download_view.py

```diff
@@ -0,0 +1,157 @@
+import argparse
+import sys
+import os
+
+import tableauserverclient as TSC
+
+EXIT_CODE_FINAL_STATUS_SUCCESS = 0
+EXIT_CODE_UNKNOWN_ERROR = 3
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_RESOURCE = 201
+EXIT_CODE_FILE_WRITE_ERROR = 1021
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--username', dest='username', required=True)
+    parser.add_argument('--password', dest='password', required=True)
+    parser.add_argument('--site-id', dest='site_id', required=True)
+    parser.add_argument('--server-url', dest='server_url', required=True)
+    parser.add_argument('--view-name', dest='view_name', required=True)
+    parser.add_argument('--file-type', choices=['image', 'thumbnail', 'pdf', 'csv'], type=str.lower, required=True)
+    parser.add_argument('--file-name', dest='file_name', required=True)
+    parser.add_argument('--file-options', dest='file_options', required=False)
+    parser.add_argument('--workbook-name', dest='workbook_name', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def authenticate_tableau(username, password, site_id, content_url):
+    """TSC library to sign in and sign out of Tableau Server and Tableau Online.
+
+    :param username:The name of the user.
+    :param password:The password of the user.
+    :param site_id: The site_id for required datasources. ex: ffc7f88a-85a7-48d5-ac03-09ef0a677280
+    :param content_url: This corresponds to the contentUrl attribute in the Tableau REST API.
+    :return: connection object
+    """
+    try:
+        tableau_auth = TSC.TableauAuth(username, password, site_id=site_id)
+        server = TSC.Server(content_url, use_server_version=True)
+        server.version = '3.15'
```

Comment:
Not necessarily asking for a change, but can you provide clarification on what setting the explicit server version does? 
Is this going to eventually break scripts that aren't using a compatible version with the servers they are trying to connect to?